README.md

```diff
@@ -5,3 +5,10 @@
 - This one is correct
 - This needs a fix
 - This one is correct too
+
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/troubleshoot-codacy/markdown-fixes/pullRequest?prid=7586272)
```suggestion

```